.github/workflows/deploy.yml

```diff
@@ -0,0 +1,52 @@
+name: Winden deployment
+
+on:
+  workflow_run:
+    workflows: transfer-repo-CI # Reuse the name of your tests workflow
+    branches: main
+    types: completed
+
+jobs:
+  deploy:
+    # `if` required because a workflow run is triggered regardless of
+    # the result of the previous workflow (see the documentation page)
+    if: ${{ github.event.workflow_run.conclusion == 'success' }}
+
+    runs-on: ubuntu-22.04
+    env:
+      TEST_ENV: foobar
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
+        with:
+          submodules: recursive
+
+      - name: setup npm, yarn etc.
+        uses: actions/setup-node@v2
+        with:
+          node-version: '16'
+
+      - name: Install docker and docker-compose
+        run: |
+          sudo curl -o /usr/local/bin/docker-compose -L "https://github.com/docker/compose/releases/download/v2.6.1/docker-compose-linux-x86_64"
+          sudo chmod +x /usr/local/bin/docker-compose
+      - name: create config
+        run: |
+          touch ./client-e2e/.env
+          cat <<EOF >> ./client/.env
+          MAILBOX_URL="wss://mailbox.winden.app/v1"
+          RELAY_URL="wss://relay.winden.app/"
+          SFTP_HOSTNAME=${{ secrets.SFTP_HOSTNAME }}
+          SFTP_USERNAME=${{ secrets.SFTP_USERNAME }}
```

Comment:
Is it possible to use keys here instead of passwords?

Reply:
That is something we already tried out, there is no option from the UI of the provider. Adding a key in `.ssh/authorized_keys` was not successful as well. The preferred option to transfer data in the UI is to use classical FTP. The username/password is solely used for the data transfer and nowhere else, so we should be able to take that risk.

Reply:
Update: `lftp` still asks for a password when using key based authentication. There can be any input and then it will use the key for initiating the connection. We use `lftp` because of its mirror feature which `sftp` does not provide. Using `rsync` is not possible as we cannot have interactive SSH access. Will give it a try.

Reply:
Hmm, okay, darn.
I guess a "medium-term" solution would be to just host the HTML ourselves -- is there any reason _not_ to do that (besides "time" in the short-term)?

Reply:
The updated solution allows to use key based auth. Just need to give an arbitrary password to avoid the prompt.